jakarta-eclipse/org.eclipse.lsp4jakarta.tests/projects/jakarta-sample/src/main/java/io/openliberty/sample/jakarta/cdi/InjectAndDisposesObservesObservesAsync.java

```diff
@@ -0,0 +1,50 @@
+package src.main.java.io.openliberty.sample.jakarta.cdi;
+
+import jakarta.inject.Inject;
+import jakarta.enterprise.inject.Disposes;
+import jakarta.enterprise.event.Observes;
+import jakarta.enterprise.event.ObservesAsync;
+
+public class InjectAndDisposesObservesObservesAsync {
+    
+    @Inject
+    public String greetDisposes(@Disposes String name) {
+        return this.greeting + " " + name + "!";
```

Comment:
`this.greeting` is not set in the class, I know it is just used for testing but we should either remove references to `this.greeting` or set it in a constructor